### ISSUE TITLE
As a user I cannot do an exact match search

### ISSUE THREAD

**Issue by djvanenckevort** [molgenis/molgenis#8254]
### How to Reproduce
- Search the BBMRI-NL catalogue bbmri_nl_persons table for the e-mail address info@bbmri.nl
- REST-API query https://catalogue.bbmri.nl/api/v2/bbmri_nl_persons?email==info%40bbmri.nl 

### Expected behavior
One matching record

### Observed behavior
299 matching records

**Comment by mswertz** [comment]
@djvanenckevort did you try
https://catalogue.bbmri.nl/api/v2/bbmri_nl_persons?q=email==info%40bbmri.nl
(notice the 'q')

**Comment by dennishendriksen** [comment]
@djvanenckevort ?

**Comment by djvanenckevort** [comment]
REST API indeed works, but in the web interface you can't, and it produces really unexpected results.

**Comment by cpavanrun** [comment]
I concur that the fuzzy search is problem within the web interface. 

We often use the Data Explorer to match identifiers or our samples, not being able to rely on the web interface to give us accurate results when querying is tricky. We end-up having to manually double check the often very similar identifiers.

**Comment by fdlk** [comment]
In 8.4, you can tag an attribute with the "token" tag.
Then a SEARCH on the field will only match values that are case-insensitive-equal to the search string.

**Comment by cpavanrun** [comment]
Are id attributes by default considered "tokens" tags?

**Comment by fdlk** [comment]
@cpavanrun no, not by default, same with enums, you have to explicitly tag them as token.
